charts/aws-ebs-csi-driver/templates/metrics.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.controller.enableMetrics -}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ebs-csi-controller
+  namespace: kube-system
+  labels:
+    app: ebs-csi-controller
+spec:
+  selector:
+    app: ebs-csi-controller
+  ports:
+    - name: metrics
+      port: 3301
+      targetPort: 3301
+  type: ClusterIP
+---
+{{- if (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: ebs-csi-controller
+  namespace: kube-system
+  labels:
+    app: ebs-csi-controller
+    release: prometheus
```

Comment:
It would be great if these labels were configurable / overridable without needing to use helm's kustomize post-renderers support.

In my setup `release` label should have a different value. Maybe `app` label can remain fixed, but `release: prometheus` one should ideally be part of e.g. `controller.serviceMonitor.additionalLabels`, to allow for overrides. Then also enable switch can be moved under `controller.serviceMonitor.enabled`

WDYT @torredil ?

Reply:
@stevo-f3 Thanks for the feedback, included this improvement in #1419.